easybuild/easyconfigs/g/googledrivedownloader/googledrivedownloader-0.4-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'googledrivedownloader'
```

Comment:
@seb45tian Wouldn't `google-drive-downloader` make more sense?

This does require changing the `source_urls` and `sources` though, and fixing the easyconfig filename, of course...

If you need help with this, do let us know!